lib/multiplexer/multiplexer_test.go

```diff
@@ -256,6 +255,48 @@ func TestMux(t *testing.T) {
 		require.NotNil(t, err)
 	})
 
+	// makes sure the connection gets dropped
+	// when PROXY protocol is 'on' but PROXY line isn't received
+	t.Run("required PROXY line", func(t *testing.T) {
```

Comment:
Can a test be added for when `proxy_protocol` is also set to `ignored`?

Reply:
I've added test in the `wrapper_test.go` that checks it, but yeah, I guess I can also add it here more explicitly.